robotd/devices.py

```diff
@@ -97,6 +97,30 @@ def status(self):
         return {'temperature': temp_milli_degrees / 1000}
 
 
+class GameState(Board):
```

Comment:
I agree with this however I think it should be fixed after the release next Monday